client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -196,18 +226,57 @@ class CampaignAdminList extends SilverStripeComponent {
           <Toolbar showBackButton handleBackButtonClick={this.props.handleBackButtonClick}>
             <Breadcrumb multiline />
           </Toolbar>
+          {newItemInfo}
           <div className={bodyClass.join(' ')}>
             {body}
           </div>
           <div className="toolbar toolbar--south">
             {this.renderButtonToolbar()}
           </div>
         </div>
-        <Preview itemLinks={itemLinks} itemId={itemId} onBack={this.handleCloseItem} />
+        {this.renderPreview(itemLinks, itemId)}
       </div>
     );
   }
 
+  renderPreview(itemLinks, itemId) {
+    let preview = null;
+
+    if (this.state.loading) {
+      preview = (
+        <div
+          className="flexbox-area-grow
+          fill-height
+          preview
+          campaign-admin__campaign-preview
+          campaign-admin__campaign-preview--empty"
+        >
+          <p>Loading...</p>
+        </div>
+      );
+    } else if (!this.getItems() || this.getItems().length === 0) {
+      preview = (
+        <div
+          className="flexbox-area-grow
+          fill-height
+          preview
+          campaign-admin__campaign-preview
```

Comment:
shouldn't have new lines in double quoted strings...
move these to a variable or classnames object